.travis.yml

```diff
@@ -0,0 +1,10 @@
+language: go
+sudo: false
+go:
+    - 1.7
+install:
+    - cd cluster-state-service; make get-deps; cd ../
+    - cd daemon-scheduler; make get-deps; cd ../
+script:
+    - cd cluster-state-service; make; make release; cd ../
```

Comment:
Not very familiar with travis. But should this include a target that actually runs the unit tests? I don't think make release runs unit tests.